scripts/build-types.ts

```diff
@@ -0,0 +1,24 @@
+import cue from 'cuelang-js';
+import { default as openapiTS } from 'openapi-typescript';
+
+import { promises as fs } from 'fs';
+
+const cueToTSTypes = async (input: string) => {
+	const cueresult = await cue('export', [input], { '--out': 'openapi' });
+	if (cueresult.code !== 0) {
+		console.error(cueresult.stderr);
+		throw new Error('Error exporting cue as openapi');
```

Comment:
Thrown error will result in call to `console.error`, above `console.error` seems redundant. Can include the `stderr` in the `Error` message. 

```
Error(`Failed to export cue to openapi: ${cueresult.stderr}`);
```

Reply:
Fixed in latest commit